src/main/cpp/autonomous/AutoNavBarrelPath.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2021 FRC Team 3512. All Rights Reserved.
+
+#include <frc/geometry/Pose2d.h>
+#include <wpi/math>
+
+#include "Robot.hpp"
+
+namespace frc3512 {
+
+void Robot::AutoNavBarrelRacing() {
+    /*
+        Initial Pose - Robot positioned near the D1 marker in the Start & Finish
+        Zone
+    */
```

Comment:
Prefer `//` over
```cpp
/*
 *
 */
```
for comments. We only use multiline comments for Doxygen docs for functions and classes, which have the form
```cpp
/**
 *
 */
```